app/controllers/admin/admin_controller.rb

```diff
@@ -3,7 +3,18 @@ class Admin::AdminController < ApplicationController
 
   def authorized?
     if current_user.nil? || !current_user.admin?
-      render plain: "404", status: 404
+      render_404
+    end
+  end
+
+  private
+
+  def render_404
+    respond_to do |format|
+      format.html { render :file => "#{Rails.root}/public/404",
+                    :layout => false, :status => :not_found }
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Expression at 15, 61 should be on its own line.<br>Use the new Ruby 1.9 hash syntax.